src/net/types.h

```diff
@@ -28,6 +28,30 @@ struct NetworkOptions {
     int port { 0 };
 };
 
+struct BaseStatus {
+    
+};
+
+// dummy structure for testing
+struct CreatureStatus : public BaseStatus {
+    CreatureStatus() : x(0), y(0), z(0), heading(0), // dummy
```

Comment:
This can be compacted into:

```cpp
struct CreatureStatus : public BaseStatus {
    CreatureStatus() = default; // redundant

    float x { 0.0f };
    double heading { 0.0 };
    uint8_t anim { 0 };
    uint16_t equipment1 { 0 };
};
```